src/apps/test/pfutest.cpp

```diff
@@ -0,0 +1,133 @@
+#include "pfu.hpp"
+#include "../test/MemFile.hpp"
+#include "../test/fileutils.hpp"
+#include "common/Configuration.hpp"
+#include "save.hpp"
+#include "string_utils.hpp"
+
+#include <catch2/catch.hpp>
+#include <fmt/format.h>
+
+extern void boot_db();
+extern void reap_old_chars();
+
+using namespace pfu;
+
+namespace {
+spdlog::logger logger = spdlog::logger("pfutest");
+}
+
+struct LoadTinyMudOnce : Catch::TestEventListenerBase {
```

Comment:
Love it! Did you consider using a Fixture instead for this? Or am I missing something? :) Does this run at a different spot than a fixutre? Or is it run only one time and you're trying to avoid taking a long time to run tests?

Reply:
Yeah, using the event listener with `testRunStarting`, it only runs once. Whereas from what I can see, specifying a test fixture class as part of TEST_CASE will create an instance of it for each case/method you specify it on. Not quite what I was looking for.  Added to that, due to the current global nature of the mud world, it'll barf if you try to load the same mud entities more than once. Fixable if we have a proper lifecycle for the mud at some point.